### DISCUSSION TITLE
Freya 5.1 bugs

### DISCUSSION THREAD

**Discussion by the-ahnam** [primefaces/community#236] [PrimeVue Templates] [answered]
I post this because I wish your products getting better, Because your design are very beautiful.

Bug 1:
In page: http://localhost:4173/#/uikit/message
the Toast not work

Bug 2:
The css files are missing, like
public\layout\styles\theme\theme-light\teal\theme.css

That will cause when a new customer installed your product, and looks like this ^_^

![WeChat Screenshot_20231014005658](https://github.com/primefaces/community/assets/26541360/c2d5661a-52ff-4bfc-bd09-564b82ef50bb)

**Comment by tugcekucukoglu** [comment, marked as answer]
1) Could you try to add Toast component to the App.vue

2) You could find public\layout\styles\theme\theme-light\teal\theme.scss and run this command to add css files

```
sass --update public/layout/:public/layout/ --no-source-map
```

It seems we forgot to add Toast component. Sorry for the confusion.

**Reply by elburro1887** [reply]
Thanks for this command. This needs to be put in your main theme documentation somewhere, I was just installing Avalon 5.2.0 for example and I had no clue why the CSS file didn't exist.

**Reply by tony-thomas-git** [reply]
I have the same problem today after purchasing and installing Ultima theme for vue. I am running in the unziped new project with no changes to the code. I made no changes and my error is not specifically related to Toast, it's an error from fresh install. And the file is at the path specified.
![image](https://github.com/primefaces/community/assets/43186848/8184d846-5702-4a5b-96a7-f0c0af8f9104)

EDIT: running the command you provided at the root of the project fixed the issue.
`sass --update public/layout/:public/layout/ --no-source-map`

**Comment by itxcore** [comment]
Hi dosent works for me just installed freya template.
Also theme.css not found
how to fix this pls asap.
sass --update public/layout/:public/layout/ --no-source-map gives me other errors see image attached.
I'm on Linux and use ruby-sass package as sass command was not found initaili.

Thank you for your help

![Uploading fira.png…]()

**Reply by itxcore** [reply]
Same on Atlantis Template

- extracted template on linux ubuntu server (node 20.x)
- npm install
- adjusted vite.conf with port and host: "0.0.0.0"
- npm run dev
- theme.css not found
